### ISSUE TITLE
Weighing the 0-1 loss to tackle imbalance and.

### ISSUE THREAD

**Issue by arvind0422** [arvind0422/Chromputer_Arvind#5]
Training with 50-50 split for ones and zero tends to overfit the ones as seen in the accuracies of 1's in the validation set.

Weighing the loss improved AUPRC scores.

**Comment by arvind0422** [comment]
[results.pdf](https://github.com/arvind0422/Chromputer_Arvind/files/2120817/results.pdf)

**Comment by arvind0422** [comment]
[auprc_scores.pdf](https://github.com/arvind0422/Chromputer_Arvind/files/2120832/auprc_scores.pdf)

This is the latest AUPRC scores that I got.
